CMakeLists_files.cmake

```diff
@@ -31,6 +31,10 @@
 list (APPEND TEST_SOURCE_FILES
 	)
 
+list(APPEND MAIN_SOURCE_FILES
+	opm/upscaling/RelPermUtils.cpp
```

Comment:
So they are private headers? I wonder if we should try to use a consistent way of treating them (location, naming) to make it obvious?